Resources/Prototypes/Entities/Objects/Weapons/Melee/e_sword.yml

```diff
@@ -50,6 +50,11 @@
         shader: unshaded
   - type: DisarmMalus
     malus: 0
+  - type: Reflect
+    chance: 0.5
+    spread: 30
```

Comment:
As is, combined with being able to offhand the esword, this is significantly too strong as it's a 50% damage reduction to ranged attacks for 8TC.

SS13 has shields so I would reduce the change to 25% and bump the spread higher as the bullets are almost undodgeable for the shooter and hit them very consistently, especially with ping.